src/ProtonPlayer.js

```diff
@@ -9,6 +9,8 @@ import Loader from './Loader';
 import Clip from './Clip';
 import { getSilenceURL } from './utils/silence';
 import initializeiOSAudioEngine from './utils/initializeiOSAudioEngine';
+import MediaSourceInterface from './interface/MediaSource';
+import AudioContextInterface from './interface/AudioContext';
```

Comment:
I'd call these 2 "engines" instead of "interfaces", becase (in typed programming languages) interfaces usually don't have an implementation tied to them.

Reply:
I know the terminology overlaps with Java, etc, but I use the word interface very deliberately in my codebases to refer to anything that bridges to some external stateful system. So `interface/HTTP`, `interface/Websocket`, etc would also exist.